target/linux/realtek/dts-5.10/rtl8380_d-link_dgs-1210-10p.dts

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x_d-link_dgs-1210_soc.dtsi"
+
```

Comment:
Why are you missing the poe include here?  The DGS-1210-10P also has PoE with a 65W budget.

Reply:
It isn't missing, it has never been defined in the original dts and have only made a 1:1 rebuild. Please see commit message part: 'Implementation:'.

Reply:
This device SoC change has been removed from the commit.